src/getExtension/index.ts

```diff
@@ -1,4 +1,4 @@
-import { ApiWindow, InjectedWindowProvider, PubSubSessionV1, PubSubSessionV2 } from '../types'
+import { ApiWindow, InjectedWindowProvider, PubSubSessionV1, PubSubSessionV2 } from '../types/index.js'
```

Comment:
```suggestion
import type { ApiWindow, InjectedWindowProvider, PubSubSessionV1, PubSubSessionV2 } from '../types/index.js'
```

Reply:
add in: [3299408](https://github.com/KILTprotocol/kilt-extension-api/pull/24/commits/329940803b8b24bcb5590bca479c6cb4d1e1011e)